packages/chakra-components/src/MenuBar/SubMenuList.js

```diff
@@ -151,35 +152,48 @@ const SubMenuList = ({
     },
     offset: { enabled: true, offset: `${skid}, ${gutter}` },
   };
-
   const styleProps = useMenuListStyle();
 
-  return (
-    <Popper
-      usePortal={false}
-      as={Comp}
-      anchorEl={titleRef.current}
-      ref={menuRef}
-      isOpen={isOpen}
-      placement={placement}
-      modifiers={popperModifiers}
-      width={width}
-      rounded="md"
-      py={2}
-      zIndex="2"
-      _focus={{
-        outline: 0,
-      }}
-      role="menu"
-      aria-label={ariaLabel}
-      tabIndex={-1}
-      onKeyDown={handleKeyDown}
-      onBlur={handleBlur}
-      {...eventHandlers}
-      {...styleProps}
-      {...props}
-    />
-  );
+  if (!isCollapsable) {
+    return (
+      <Popper
+        usePortal={false}
+        as={Comp}
+        anchorEl={titleRef.current}
+        ref={menuRef}
+        isOpen={isOpen}
+        placement={placement}
+        modifiers={popperModifiers}
+        width={width}
+        rounded="md"
+        py={2}
+        zIndex="2"
+        _focus={{
+          outline: 0,
+        }}
+        role="menu"
+        aria-label={ariaLabel}
+        tabIndex={-1}
+        onKeyDown={handleKeyDown}
+        onBlur={handleBlur}
+        {...eventHandlers}
+        {...styleProps}
+        {...props}
+      />
+    );
+  } else {
+    return (
+      <Box
+        as="ul"
+        ref={menuRef}
+        width={width}
+        display={isOpen ? "block" : "none"}
+        onKeyDown={handleKeyDown}
+        {...eventHandlers}
+        {...props}
+      />
+    );
+  }
```

Comment:
```suggestion
  if (isCollapsable) {
    return (
      <Box
        as="ul"
        ref={menuRef}
        width={width}
        display={isOpen ? "block" : "none"}
        onKeyDown={handleKeyDown}
        {...eventHandlers}
        {...props}
      />
    );
  }
    return (
      <Popper
        usePortal={false}
        as={Comp}
        anchorEl={titleRef.current}
        ref={menuRef}
        isOpen={isOpen}
        placement={placement}
        modifiers={popperModifiers}
        width={width}
        rounded="md"
        py={2}
        zIndex="2"
        _focus={{
          outline: 0,
        }}
        role="menu"
        aria-label={ariaLabel}
        tabIndex={-1}
        onKeyDown={handleKeyDown}
        onBlur={handleBlur}
        {...eventHandlers}
        {...styleProps}
        {...props}
      />
    );
```

Reply:
Make this change and add the create a collapsable style object for placement in Popper.

```js
if(isCollapsable === true) {
  collapsablePopperProp: {
    placement: 'right'
  }
}

this will override the default `placement: 'bottom'`